maud_macros/src/generate.rs

```diff
@@ -48,10 +48,10 @@ impl Generator {
                 }
             },
             Markup::Literal { content, .. } => build.push_escaped(&content),
-            Markup::Symbol { symbol } => self.name(symbol, build),
-            Markup::Splice { expr, .. } => build.push_tokens(self.splice(expr)),
-            Markup::Element { name, attrs, body } => self.element(name, attrs, body, build),
-            Markup::Let { tokens, .. } => build.push_tokens(tokens),
+            Markup::Symbol { symbol } => self.name(symbol.into(), build),
+            Markup::Splice { expr, .. } => build.push_tokens(self.splice(expr.into())),
+            Markup::Element { name, attrs, body } => self.element(name.into(), attrs, body, build),
+            Markup::Let { tokens, .. } => build.push_tokens(TokenStream::from(tokens)),
```

Comment:
Can we change `push_tokens` to accept `TokenStream` directly instead of `impl IntoIterator<...>`? Then this line can be just `tokens.into()` instead.